client/src/index.html

```diff
@@ -22,6 +22,71 @@
     <div class="blob blob-4"></div>
     <div class="blob blob-5"></div>
   </div>
+  
+  <script type="application/ld+json">
+{  
+  "@context": "http://schema.org",
+  "@type":"VideoGame",
+  "name":[
+    {
+      "@language":"en",
+      "@value":"Momentum Mod"
+    }
+  ],
+ 
+  "description":[
+    {
+      "@language":"en",
+      "@value":"Momentum Mod is a standalone game built on the Source Engine, aiming to centralize movement gametypes found in CS:S, CS:GO, and TF2."
+    }
+  ],
+  "genre":[
+    "Platformer",
+    "Third-person shooter"
+  ],
+  "url":"https://momentum-mod.org/",
+  "applicationCategory": "Game",
+  "operatingSystem": "Windows, Mac & Linux",
+  "image":"https://momentum-mod.org/assets/images/site.png",
+  "screenshot":"https://momentum-mod.org/assets/images/gamemode_surf.jpg",
+  "aggregateRating": {
+    "@type": "AggregateRating",
+    "ratingValue": "1",
+    "reviewCount": "1"
+  },
+  "sameAs": [
+    "https://docs.momentum-mod.org/",
+    "https://www.youtube.com/channel/UCCGfc_79o7hUDFodxpNu9_Q"
+  ],
+  "trailer":{
+    "@type":"VideoObject",
+    "url":"https://www.youtube.com/watch?v=RnTVMuZ5Fg8",
+    "thumbnailUrl":"https://i.gyazo.com/c558a18ff8bcf7e2eb1aa5538cb85eca.png",
+    "uploadDate":"06/05/2019",
+    "inLanguage":"en",
+      "description":[
+    {
+      "@language":"en",
+      "@value":"What is Momentum Mod?"
+    }
+  ],
+    "name":[
+    {
+      "@language":"en",
+      "@value":"Momentum Mod Tutorial Video"
+    }
+  ]
+  },
+  "exampleOfWork":[
```

Comment:
What is the purpose of the exampleOfWork here? it causes errors on the testing tool and doesn't create a proper VideoGame. From what I read of the docs this is supposed to be a game that momentum mod derives from, should it be CS:S or another source game?